packages/design-preview/src/components/style.scss

```diff
@@ -24,6 +24,7 @@ $break-design-preview: 1024px;
 
 .design-preview__sidebar {
 	align-items: center;
+	background: #fdfdfd;
```

Comment:
It might be better to reset to `transparent` when the sidebar is not sticky on the top 🙂

Reply:
Because it is done in a mobile-first way, this is the simplest approach because the sidebar sticks to the top by default. Maybe I am missing something.

Reply:
Sorry for not being clear,

> Is there a colour variable you could use here? That way CSS is more maintainable and Calypso colour themes would continue working as expected.

I mean setting the background to transparent in desktop env so that the color would be the same as the body. Referring to https://github.com/Automattic/wp-calypso/pull/70229#discussion_r1027880391, we don't have a colour variable for it, so the background color of the body might be changed (rarely)

Or we could define something like `--color-body-background: #fdfdfd` and use it in both places. What do you think?

Reply:
I'd prefer to add the variable. Should we add it in "@automattic/calypso-color-schemes"?